src/pickers/_datepicker.scss

```diff
@@ -0,0 +1,343 @@
+.mdl-datepicker {
+  * {
+    box-sizing: border-box;
```

Comment:
Consider adding this to each of the selectors below instead (or just the ones that need it, even).